### ISSUE TITLE
Reaching out

### ISSUE THREAD

**Issue by cpod12** [xunkar/Grids#1]
Hey, I just wanted to reach out again and say the stuff you're doing is really cool, and was wondering if you'd like any help with the local AI service you're working on? I'd love to help in any way I can since what I've seen from the demo is really impressive

**Comment by cpod12** [comment]
Sorry about reaching out on here btw. Just realized you had an email and discord listed rn

**Comment by xunkar** [comment]
To be honest I had put this project on hold since I wasn't getting any update on the RetroArch PR, maybe now it will be merged thanks to your efforts? Anyways, my personal take on the translation tool isn't about using AI to run the translation, just so we're clear. "AI service" is a very misguiding name that RetroArch uses for this, and they're not going to change it. Of course once the PR is merged it becomes totally possible to use AI to run the translation, but it can be used for many different things. The idea behind the PR was that the code in RetroArch was performing very poorly though, making it impossible to run the service automatically without slowing down games. 

But my software would still only provide manual translations. The goal was to offer a complete text editor for translators to make fan translations that would work like patches, without the need to patch anything, while also getting rid of technical constraints like ROM size and fonts etc. I also don't really believe in (so-called) AI for translation purposes, as it's not an objective truth and the dataset you'd use to train such an algorithm would be inconsistent. But I'm sure it would also have its use, and it might be better than nothing. I have included all the necessary documentation for RA's AI service in the PR (it's in the translation service header file) if you'd rather tackle this particular challenge.

Otherwise, you know where to find me :)

**Comment by cpod12** [comment]
Ok, thank you so much! I'll probably reach out to you in the future if I go down that route :)
